packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md

```diff
@@ -1,25 +1,10 @@
-### 1: Build a sign-in page on the client
```

Comment:
Consider moving to the requirements. This step doesn't have any instructions and seems like more of a prerequisite. The image doesn't offer any additional content and can be removed.